.github/workflows/relayrunner-pr.yml

```diff
@@ -0,0 +1,73 @@
+name: Build RelayRunner-PR
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, closed]
+
+    paths:
+      - 'Dockerfile'
+      - '**.js'
+      - '**.css'
+      - '.github/workflows/relayrunner-pr.yaml'
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_REPO: abassey2/relayrunner
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Login to Container Registry
+      uses: docker/login-action@v1
+      with:

```

Comment:
use this to push to retaildevcrews/relayrunner

registry: ghcr.io
username: ${{ secrets.GHCR_ID }}
password: ${{ secrets.GHCR_PAT }}

GHCR_* are "organization" secrets